README.md

```diff
@@ -16,72 +16,72 @@ Backed by Y Combinator
 
 ---
 
-**Fern is an open source format for defining REST APIs.**
-You can think of it like a programming language to describe
-your API: your endpoints, types, errors, and examples.
+# Fern
 
-This repository contains the **Fern compiler.** The compiler transforms the API description into useful outputs, like:
+**Fern** is an **open source toolkit** for defining, building and generating **APIs, documentation and SDKs**. Bring an existing OpenAPI definition or describe your endpoints in a Fern specification. Fern generates backend code, specifications in various formats, and client libraries with a single command.
 
-### 🌿 SDKs
+This repository contains the Fern **compiler**. The compiler transforms an API definition into various code artifacts, such as client libraries, SDKs, or even full-fledged **Express**, **Fast API** or **Spring Boot** backends.
 
-Client libraries speed up internal development, and help acquire customers who use your API. Our auto-generated SDKs are idiomatic and feel handwritten.
+In short, Fern takes care of the boilerplate associated with developing APIs, so you can **focus on delivering business value**.
 
-### 🌿 Server-side code generation
+# Capabilities
 
-We automatically generate lots of boilerplate on the server side, like Pydantic models for FastAPI and Jersey interfaces for Spring Boot. We also add compile-time validation that all your endpoints are being served correctly.
+Currently, the Fern compiler can generate the following types of artifacts:
 
-### 🌿 Postman Collection
+### 🌿 Client Libraries and SDKs
 
-Complete with examples of successful and unsuccessful requests!
+Fern generates idiomatic SDKs you can use to interact with your APIs, or publish for third party usage. This makes keeping your client libraries up to date with your API endpoints trivial, and speeds up your workflows dramatically.
 
-### 🌿 An OpenAPI spec
+### 🌿 Backend Code Generation
 
-You can feed the generated OpenAPI into the endless list of tools that support OpenAPI.
+Define your API, and Fern will generate the boilerplate for you. All the generated code is validated at build time, ensuring your all your endpoints and their data are being served correctly.
```

Comment:
would be specific, such as mentioning Fern will generate the models and networking logic. Just add your business logic (also called application logic by devs, not sure which term is better)

Reply:
I think business logic is the broader term, at least in DDD and such. I reckon "endpoints" is more applicable than networking logic here, because REST APIs rely on HTTP for their networking, we're just implementing an endpoint.